.github/CODEOWNERS

```diff
@@ -0,0 +1,20 @@
+# Copied files:
+
+.github/dco.yml @IBM/cloudant-sdks
+scripts/setup_couch.sh @IBM/cloudant-sdks
+scripts/setup_wiremock.sh @IBM/cloudant-sdks
+stubs/mappings.json @IBM/cloudant-sdks
+KNOWN_ISSUES.md @IBM/cloudant-sdks
+Jenkinsfile @IBM/cloudant-sdks
+
+# Generated code files:
+
+cloudant/v1.ts @IBM/cloudant-sdks
+test/unit/cloudant.v1.test.js @IBM/cloudant-sdks
+
+# Bump version files:
+
+.bumpversion.cfg @IBM/cloudant-sdks
+# README.md
```

Comment:
I suggest we remove the commented lines.  Same suggestion in https://github.com/IBM/cloudant-java-sdk/pull/47#discussion_r528755420

Reply:
As we agreed with @ricellis offline. The package files will stay uncommented, README does not, till it is not generated too.